### PR TITLE
Upwork.com changes (update 1)

### PR DESCRIPTION
**Pull request by Mirm0n**
So I posted here earlier about some upwork.com fixes.

Here they are.
before-after-contrast-change
Untitled-1

These are screenshots of the main fixes.
I'll be adding more fixes in the future, but now I'm busy with something.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -17100,6 +17100,64 @@ CSS
 
 ================================
 
+upwork.com
+/*---------------Home page fixes-------------------*/
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17104]: Please remove comment, they aren't allowed to be in the fixes file

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -17100,6 +17100,64 @@ CSS
...
+/* fix 1 Remove drop shadow of green button in right column */
+
+CSS
+.profile-completeness-nudges-tiles-alternative .carousel-wrapper .up-icon svg {
+ filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white));
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17109]:
```suggestion
    filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white)); 
```
Use 4 spaces

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -17100,6 +17100,64 @@ CSS
...
+/*---------------Home page fixes-------------------*/
+/* fix 1 Remove drop shadow of green button in right column */
+
+CSS
+.profile-completeness-nudges-tiles-alternative .carousel-wrapper .up-icon svg {
+ filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white)); 
+}
+/* ------fix 2 REMOVE WHITE GRADIENT HIGHLIGHTS------ */
+.up-skill-container .up-btn.up-btn-next::before { opacity: 0; }
+.up-skill-container .up-btn.up-btn-prev::before { opacity: 0; }
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17113]:
```suggestion
.up-skill-container .up-btn.up-btn-next::before,
.up-skill-container .up-btn.up-btn-prev::before {
    opacity: 0 !important;
}
```

Use the following style:
```
selector {
    CSS property: CSS value !important
}
```

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -17100,6 +17100,64 @@ CSS
...
+    
+}
+.layout-container {
+    
+    background-color: #1f2223;
+}
+.layout-container {
+    
+    height: 430%;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17135]: This seems like a cosmetic fix, so it doesn't fix a faulty behavior from Dark Reader. But rather a opinionated fix.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Mirm0n) [reply src/config/dynamic-theme-fixes.config]: is that correct

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Yes.

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (Mirm0n) [reply src/config/dynamic-theme-fixes.config]: I suppose it's resolved now right?

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (Mirm0n) [reply src/config/dynamic-theme-fixes.config]: Removed it

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Are there more of such fixes? I'm not really able to tell, some seem to be indeed fixing faulty behavior, but other just seems like personal preferences.

Comment (Mirm0n) [reply src/config/dynamic-theme-fixes.config]: Hold on. I'll remove some stuff now. There was some things I was suppose to remove but forgot

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -17100,6 +17100,28 @@ CSS
...
+CSS
+.profile-completeness-nudges-tiles-alternative .carousel-wrapper .up-icon svg {
+    filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white)); 
+}
+
+.up-skill-container .up-btn.up-btn-next::before,
+.up-skill-container .up-btn.up-btn-prev::before {
+    opacity: 0 !important;
+}
+.up-tab-scroll-hint:after { background: none;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17114]:
```suggestion
.up-tab-scroll-hint:after {
    background: none !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 6]
```diff
@@ -17100,6 +17100,28 @@ CSS
...
+}
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17116]:
```suggestion
```

File: src/config/dynamic-theme-fixes.config [hunk 7]
```diff
@@ -17100,6 +17100,28 @@ CSS
...
+.up-d-skeleton {
+    color: transparent !important;
+    background-color: var(--darkreader-bg--skeleton-color)  !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17119]:
```suggestion
    background-color: var(--darkreader-bg--skeleton-color) !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 8]
```diff
@@ -17100,6 +17100,28 @@ CSS
...
+    background-image: linear-gradient(90deg,var(--darkreader-bg--skeleton-color),#24242400,var(--darkreader-bg--skeleton-color))  !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17120]:
```suggestion
    background-image: linear-gradient(90deg, var(--darkreader-bg--skeleton-color), #24242400, var(--darkreader-bg--skeleton-color)) !important;
```

**Review by Gusted — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 9]
```diff
@@ -17100,6 +17100,28 @@ CSS
 
 ================================
 
+upwork.com
+
+CSS
+.profile-completeness-nudges-tiles-alternative .carousel-wrapper .up-icon svg {
+    filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white)); 
+}
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17109]:
```suggestion
```

**Comment by Mirm0n** [comment]
Why are the checks failing? Did I do something wrong? (Again sorry cause I'm new to this)

File: src/config/dynamic-theme-fixes.config [hunk 10]
```diff
@@ -17100,6 +17100,27 @@ CSS
...
+.up-tab-scroll-hint:after {
+    background: none !important;
+}
+.up-d-skeleton {
+    color: transparent !important;
+    background-color: var(--darkreader-bg--skeleton-color) !important;
+    background-image: linear-gradient(90deg, var(--darkreader-bg--skeleton-color), #24242400, var(--darkreader-bg--skeleton-color)) !important;
+}
+
+ ================================
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17122]:
```suggestion
================================
```

File: src/config/dynamic-theme-fixes.config [hunk 11]
```diff
@@ -17100,6 +17100,27 @@ CSS
...
+    background: none !important;
+}
+.up-d-skeleton {
+    color: transparent !important;
+    background-color: var(--darkreader-bg--skeleton-color) !important;
+    background-image: linear-gradient(90deg, var(--darkreader-bg--skeleton-color), #24242400, var(--darkreader-bg--skeleton-color)) !important;
+}
+
+================================
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17123]:
```suggestion
upwork.com

CSS
.profile-completeness-nudges-tiles-alternative .carousel-wrapper .up-icon svg {
    filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white)); 
}
.up-skill-container .up-btn.up-btn-next::before,
.up-skill-container .up-btn.up-btn-prev::before {
    opacity: 0 !important;
}
.up-tab-scroll-hint:after {
    background: none !important;
}
.up-d-skeleton {
    color: transparent !important;
    background-color: var(--darkreader-bg--skeleton-color) !important;
    background-image: linear-gradient(90deg, var(--darkreader-bg--skeleton-color), #24242400, var(--darkreader-bg--skeleton-color)) !important;
}

================================

```

**Comment by Gusted** [comment]
Some whitespaces I asssume.

**Comment by Mirm0n** [comment]
@Gusted I have a question. it doesn't seem that the fix has been applied to the extension itself. I see it on github but when I press the dev tools I don't see the upwork.com part. Is that an issue on my side? I reinstalled the extension with no luck.

**Comment by Gusted** [comment]
Please see https://github.com/darkreader/darkreader#site-fixes

**Pull request merged**